backends/openvino/quantizer/quantizer.py

```diff
@@ -116,18 +120,35 @@ def __init__(
                 preset=preset, model_type=model_type, **kwargs
             )
         else:
-            compression_mode = mode.value.replace(
-                "wo", ""
-            )  # Mode value has to match NNCF CompressWeightsMode
+            compression_mode = OpenVINOQuantizer.WEIGHTS_ONLY_COMPRESSION_MODES[mode]  # Mode value has to match NNCF CompressWeightsMode
             weight_compression_configuration = get_weight_compression_configuration(
                 nncf.CompressWeightsMode(compression_mode),
                 **kwargs,
             )
-            subset_size = 1  # Doesn't really matter in this case since it is data-free. Should just be +ve
+            weight_compression_configuration["subset_size"] = 1  # Doesn't really matter in this case since it is data-free. Should just be +ve
             self._algo = nncf.quantization.algorithms.weight_compression.algorithm.WeightCompression(
-                subset_size=subset_size, **weight_compression_configuration
+                **weight_compression_configuration
             )
 
+    def get_quantizer_specific_internal_algo_config(self) -> Dict[str, Any]:
+        """
+        Returns the config attributes like all_layers, group_size, backup_mode and Quantization mode
+        from the quantizer's internal algorithm. These attributes are defined when initializing the quantizer
```

Comment:
It is not yet a data class which would specify the interface for compress_pt2e, but let's describe all the params in the output dict. I believe this would help on the code review
```suggestion
        Returns a dictionary with all_layers, group_size, backup_mode and Quantization mode parameters
        used by the compress_pt2e weight compression algorithm. 
        * all_layers - desc...
        * group_size - desc..
        ...
        ```

Reply:
Done